Dockerfile

```diff
@@ -141,9 +154,9 @@ EXPOSE 5000
 # multiple statements using shell commands (& and &&).
 # see https://stackoverflow.com/questions/46797348/docker-cmd-exec-form-for-multiple-command-execution
 # FIXME: Test if this works using `mtcagent run "$DEMO_DIR/agent.cfg"` instead of running `cd $DEMO_DIR && mtcagent run agent.cfg`.
```

Comment:
Remove this line, as it was already determined to be working.

Reply:
cleaned up